.bazelrc

```diff
@@ -20,6 +20,7 @@ fetch:darwin --repository_cache=.bazel-cache/repo
 sync:darwin --repository_cache=.bazel-cache/repo
 
 test:oracle --repo_env DAML_ORACLE_TESTING=true --test_env ORACLE_USERNAME --test_env ORACLE_PORT --test_env ORACLE_PWD
+test:postgres  --test_env POSTGRESQL_HOST --test_env POSTGRESQL_PORT --test_env POSTGRESQL_USERNAME --test_env POSTGRESQL_PASSWORD
```

Comment:
I find it suspicious that you are the first who needs this. Is it because you wanted to look at you local DB after testing? Can it be that this is a one-off and does not warrant a change in here?
For oracle it obviously matters much more since no ephemeral oracle possible.
On any rate this change is not heavy, but not sure of the consequences: @stefanobaghino-da can you maybe chime in? WDYT?

Reply:
1. How does this depend on #13492?
2. Why do you need a non-ephemeral PostgreSQL instance?
3. How is one supposed to use this? How is this documented? How can people be aware of this and when and how to use it?
4. Is this supposed to be used as part of CI? How do you ensure that the proper environment is set up properly on our CI nodes?

Reply:
1) Removed that depedency.
2) I'm breakpointing in a test and inspect the contents of the index db
3) I updated the PR's and commit's description with my usage pattern. Happy to add to the appropriate docs in that makes sense.
4) I'm using in local environment only. I've set up the correct values in my `.envrc.private`

Reply:
> I'm breakpointing in a test and inspect the contents of the index db

Sounds like this should be a test on its own. Is there a reason not to make this manual inspection a test?

> I'm using in local environment only. I've set up the correct values in my .envrc.private

If this is a tool you use, you can make it part of your local Bazel configuration. Do you have recommendations on how to use it in other parts of the codebase?

Reply:
Ah, I can put in my `.bazelrc.local`
That seems to work and I have no incentive to push for this PR now